tests/appsec/iast_packages/packages/pkg_requests.py

```diff
@@ -0,0 +1,23 @@
+"""
+requests==2.31.0
```

Comment:
How are we making sure this is the version we're actually testing?

Do we have some hatch configuration to set the dependencies versions?

Reply:
No, the configuration and package version are the riot conf now, as I said, this PR introduces base structure to build packages tests. The next PR will introduce the package & version management :+1: